test/k8s/datapath_configuration.go

```diff
@@ -1000,6 +1000,9 @@ func testPodHTTPToOutside(kubectl *helpers.Kubectl, outsideURL string, expectNod
 	deploymentManager.WaitUntilReady()
 
 	label := "zgroup=testDSClient"
+	err := kubectl.WaitforPods(namespace, "-l "+label, helpers.HelperTimeout)
```

Comment:
Does it make sense just to wait for the daemonset itself?
Otherwise this seems fine.

Reply:
Unfortunately, waiting for the DS doesn't guarantee that its pods will be in ready state.

Reply:
Oops, you’re right. I meant `kubectl rollout`

Reply:
Yeah, `kubectl rollout` is a good idea. However, considering that we are planning to `rm $GINKGO`, I am reluctant to keep extending it.

Reply:
Makes sense.